docs/release-notes/1.5.0-preview2/release-1.5.0-preview2.md

```diff
@@ -0,0 +1,64 @@
+# [ML.NET](http://dot.net/ml) 1.5.0-preview2
+
+## **New Features (IN-PREVIEW, please provide feedback)**
+- **TimeSeriesImputer** ([#4623](https://github.com/dotnet/machinelearning/pull/4623))
+- **LDSVM Trainer** ([#4060](https://github.com/dotnet/machinelearning/pull/4060))
+- **Export-to-ONNX for below components:**
+    - SlotsDroppingTransformer ([#4562](https://github.com/dotnet/machinelearning/pull/4562))
+    - ColumnSelectingTransformer ([#4590](https://github.com/dotnet/machinelearning/pull/4590))
+    - VectorWhiteningTransformer ([#4577](https://github.com/dotnet/machinelearning/pull/4577))
+    - NaiveBayesMulticlassTrainer ([#4636](https://github.com/dotnet/machinelearning/pull/4636))
+  	- PlattCalibratorTransformer ([#4699](https://github.com/dotnet/machinelearning/pull/4699))
+
+  
+## **Bug Fixes**
+- Fix for OneVersusAllTrainer([#4698](https://github.com/dotnet/machinelearning/pull/4698))
+- Added support to run PFI on uncalibrated binary classification models ([#4587](https://github.com/dotnet/machinelearning/pull/4587))
+- Fix bug in WordBagEstimator when training on empty data ([#4696](https://github.com/dotnet/machinelearning/pull/4696))
+- Added Cancellation mechanism to Image Classification (through the experimental nuget) (fixes #4632) ([#4650](https://github.com/dotnet/machinelearning/pull/4650))
+-	Changed F1 score to return 0 instead of NaN when Precision + Recall is 0 ([#4674](https://github.com/dotnet/machinelearning/pull/4674))
+- TextLoader, BinaryLoader and SvmLightLoader now check the existence of the input file before training ([#4665](https://github.com/dotnet/machinelearning/pull/4665))
+- Changed onnx export to append a .onnx string to column names to support better mapping to ML.NET output column names ([#4734](https://github.com/dotnet/machinelearning/pull/4734))
+-	TextLoader, BinaryLoader and SvmLightLoader now check the existence of the input file before training ([#4665](https://github.com/dotnet/machinelearning/pull/4665))
+- ImageClassificationTrainer now checks the existence of input folder before training ([#4691]( https://github.com/dotnet/machinelearning/pull/4691))
+- Sweep Trimming of Whitespace in AutoML ([#3918](https://github.com/dotnet/machinelearning/pull/3918))
+- Use random file name for AutoML experiment folder ([#4657](https://github.com/dotnet/machinelearning/pull/4657))
+
+
+## **Enhancements**
+- Added in support for System.DateTime type for the DateTimeTransformer ([#4661](https://github.com/dotnet/machinelearning/pull/4661))
+- Additional changes to ExpressionTransformer ([#4614](https://github.com/dotnet/machinelearning/pull/4614))
+
+## **Test updates**
```

Comment:
This whole section is not shipped to the users, why is this included in release notes? Who is your target audience and why would they care about stuff that is not shipped part of nuget?